backend/src/auto_boundaries.rs

```diff
@@ -53,7 +53,11 @@ impl MapModel {
             severances.push(linestring.clone());
         }
 
-        for polygon in split_polygon(self.mercator.to_mercator(&self.boundary_wgs84), severances) {
+        let boundary_mercator = self.mercator.to_mercator(&self.boundary_wgs84);
+        for polygon in boundary_mercator
```

Comment:
It's a bit slow, but it works!

![image](https://github.com/user-attachments/assets/3e986aef-6913-4435-9f49-a9b4e1324b4f)